File: test/end-to-end/String Indices/string indices.js
//
// Initialization
let haystack = "Hello, GLS!";

// Searching
let needle = "GLS";
let firstIndexOf = haystack.indexOf(needle);
let secondIndexOf = haystack.indexOf(needle, firstIndexOf + needle.length);

// Results
console.log(`Found a first result at: ${firstIndexOf}.`);

if (secondIndexOf !== -1) {
    console.log(`Found a second result at: ${secondIndexOf}.`);
}
//

File: test/end-to-end/String Indices/string indices.cs
//
using System;

// Initialization
string haystack = "Hello, GLS!";

// Searching
string needle = "GLS";
int firstIndexOf = haystack.IndexOf(needle);
int secondIndexOf = haystack.IndexOf(needle, firstIndexOf + needle.Length);

// Results
Console.WriteLine(string.Format("Found a first result at: {0}.", firstIndexOf));

if (secondIndexOf != -1)
{
    Console.WriteLine(string.Format("Found a second result at: {0}.", secondIndexOf));
}
//
